kolibri/plugins/coach/assets/src/views/common/notifications/NotificationCard.vue

```diff
@@ -19,11 +19,11 @@
             <KRouterLink
               v-if="targetPage && targetPage.name"
               :text="linkText"
-              :to="$router.getRoute(targetPage.name, targetPage.params, targetPage.query)"
+              :to="getRoute(targetPage)"
             />
-            <template v-else>
+            <span v-else>
               {{ linkText }}
-            </template>
+            </span>
```

Comment:
this is fine, just curious why we added the span?

Reply:
To make it easier to target in the test